### DISCUSSION TITLE
`stale-issue-message = empty string`: Not comment with a message, or not mark as stale?

### DISCUSSION THREAD

**Discussion by janpio** [actions/stale#1093] [Q&A]
Docs are not super clear to me:

> stale-issue-message
> The message that will be added as a comment to the issues when the stale workflow marks it automatically as stale with a label.
>
> **You can skip the comment sending by passing an empty string.**
>
> Default value: unset
> Required Permission: issues: write

But then the first Usage example:

```
...
      - uses: actions/stale@v8
        with:
          stale-issue-message: 'Message to comment on stale issues. If none provided, will not mark issues stale'
          stale-pr-message: 'Message to comment on stale PRs. If none provided, will not mark PRs stale'
...
```

Is it "skip comment" or "will not mark issues stale"?
Or is the difference between "empty string" vs. "not provided"?

(Context, we have a super old fork of the action that adds an option to disable commenting, but I am not sure if that is even still needed: https://github.com/actions/stale/compare/main...pantharshit00:stale:main)

**Comment by 007mehrdad** [comment]
ocs
